src/components/PopupDelete.js

```diff
@@ -0,0 +1,24 @@
+import Popup from "./Popup.js";
+
+export default class PopupDelete extends Popup {
+  constructor(popupSelector, handleDeleteConfirm) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._handleDeleteConfirm = handleDeleteConfirm;
```

Comment:
Needs correction: you should not provide the confirm handler via the constructor arguments, since you should have only one instance of this class. Create a setter for the action and use it before opening the popup.